editor/editor_plugin.cpp

```diff
@@ -156,6 +156,11 @@ Control *EditorInterface::get_editor_viewport() {
 	return EditorNode::get_singleton()->get_viewport();
 }
 
+Node3DEditorViewport *EditorInterface::get_node3D_editor_viewport(int p_idx) const{
```

Comment:
The convention is to use `snake_case` for method signatures.

```suggestion
Node3DEditorViewport *EditorInterface::get_node_3d_editor_viewport(int p_idx) const{
```

Reply:
Gotcha. I'll get that fixed.